pkgs/tools/misc/zsh-history-to-fish/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonApplication, fetchPypi, click }:
+
+buildPythonApplication rec {
```

```suggestion
python3.pkgs.buildPythonApplication rec {
```
Usually, we import python3 at the top level and use python3.pkgs.